src/llvm

```diff
@@ -1 +1 @@
-Subproject commit 8cbcdf1b72e1b23679646f6faca265f76b20d379
+Subproject commit bff69076975642c64e76dbeaa53476bfa7212086
```

Comment:
Okay. Yeah... that merge commit is pretty nasty looking. I wonder if a rebase and squash would take care of it (and then revert the changes to "src/llvm" and "src/rust-installer")